d2cli/main.go

```diff
@@ -83,6 +83,10 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	browserFlag, err := ms.Opts.Bool("BROWSER", "browser", "", true, "open the browser and go to the address that watch streams updates to")
```

Comment:
Could you make this a string and interpet exactly like `$BROWSER`? So you'd just `ms.Env.Setenv("BROWSER", * browserFlag)` if the flag was set.

Reply:
all this is doing is validation though. it'll take 1 too. it's just helping users make less mistakes

Reply:
But browser can be used to point to a specific executable too. So --browser and BROWSER aren't equal which doesn't seem good to me.

Reply:
oh, i didn't know that. okay